src/databricks/labs/lsql/dashboards.py

```diff
@@ -1018,3 +1019,36 @@ def get_url(self, dashboard_id: str) -> str:
         # The /published redirects to the draft if the dashboard is not published
         dashboard_url = f"{self._ws.config.host}/dashboardsv3/{dashboard_id}/published"
         return dashboard_url
+
+
+class ExportDashboard(DashboardMetadata):
+    def __init__(self, sql_backend, config):
+        self._sql_backend = sql_backend
+        self._config = config
+        super().__init__(display_name="")
+
+    def export_dashboard(
+        self,
+        queries_path: Path,
+        export_path: Path,
+        catalog: str = "hive_metastore",
+        database: None = None,
+    ) -> None:
+        """Export the dashboard queries to CSV files."""
+        if database is None:
+            database = self._config.inventory_database
```

Comment:
what type is the `config`?

Reply:
This will be removed. I was thinking about the usage we want under UCX, but this has to be flexible and scalable to be used as part of any implementation.